nestkernel/exceptions.h

```diff
@@ -939,6 +939,34 @@ class KeyError : public KernelException
   std::string message() const;
 };
 
+/**
+ * Exception to be thrown if an internal error occures.
+ * @ingroup KernelExceptions
+*/
+class InternalError : public KernelException
```

Comment:
why do we need this new exception? wouldn't a `KernelException` by sufficient?

Reply:
Are not all exceptions derived from the base class `KernelException`? The enumeration that I introduced in `targets.h` as well as the `InternalError` exception should make the code more robust and easier to extend - just in case that there is more than a single status bit needed in future.